### DISCUSSION TITLE
Welcome to Skylicht Engine Discussions!

### DISCUSSION THREAD

**Discussion by ducphamhong** [skylicht-lab/skylicht-engine#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sahi2l** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 
 Hello sir 

*Very hard I got your gmail and you are not replying us on other social media platforms. If you really see our gmail then please Reply* 


This game is not even on play store(GANG WAR MAFIA) .  Nor are any of its servers running.  this game is completely closed

Can you tell me when the new format of this game will be released on play store

This is my only favorite FPS game and the only favorite of many people with me


*I am attaching some sacreenshot together so that you will understand.  what i want to say*


Tranclet in Vietnamese


Xin chào ông


 * Rất khó, tôi đã nhận được gmail của bạn và bạn không trả lời chúng tôi trên các nền tảng mạng xã hội khác.  Nếu bạn thực sự thấy gmail của chúng tôi thì hãy Trả lời *



 Trò chơi này thậm chí không có trên cửa hàng chơi (GANG WAR MAFIA).  Cũng không có bất kỳ máy chủ nào của nó đang chạy.  trò chơi này đã hoàn toàn đóng cửa


 Bạn có thể cho tôi biết khi nào định dạng mới của trò chơi này sẽ được phát hành trên Play Store không


 Đây là trò chơi FPS yêu thích duy nhất của và là trò chơi yêu thích duy nhất của nhiều người cùng tôi

**Comment by Sahi2l** [comment]
@ducphamhong 
hello sir how are you
 i hope you are well ....

I have some questions and you have to answer them !

You have to tell how much you have prepared the engine of the new game.

When will you come out with the demo of the new game?

-:my friend and i have a complaint with you.  We want You can restart the old game server. ( If you can do this then… otherwise we have no problem)
Until a new one arrives!
  
*Life was very sad without the gang war mafia*.

 *I'm literally not playing any games right now*.

I can see your hard work.  You're working a lot on the git_hub.

NOTE-: We just want a new game.  Preference should be given to G.W.M 2.0

Thank you

Sincerely 

Sahil Bishnoi &  all my friends



Translate in Vietnamese

xin chào sếp, bạn có khỏe không
  tôi hy vọng bạn khỏe ....

 Tôi có một số câu hỏi và bạn phải trả lời chúng!

 Bạn phải cho biết bạn đã chuẩn bị bao nhiêu cho động cơ của trò chơi mới.

 Khi nào bạn sẽ ra bản demo của trò chơi mới?

 -: bạn của tôi và tôi có khiếu nại với bạn.  Chúng tôi muốn Bạn có thể khởi động lại máy chủ trò chơi cũ.  (Nếu bạn có thể làm điều này thì… nếu không thì chúng tôi không có vấn đề gì)
 Cho đến khi một cái mới đến!
  
 * Cuộc sống thật buồn khi không có băng đảng mafia *.

  * Tôi thực sự không chơi bất kỳ trò chơi nào ngay bây giờ *.

 Tôi có thể thấy sự chăm chỉ của bạn.  Bạn đang làm việc rất nhiều trên git_hub.

 LƯU Ý-: Chúng tôi chỉ muốn một trò chơi mới.  Ưu tiên nên dành cho G.W.M 2.0

 Cảm ơn bạn

 Trân trọng

 Sahil Bishnoi và tất cả bạn bè của tôi

@ducphamhong

**Comment by CarverNZ** [comment]
Thank you Sir!
I've been using Unity for a while but have really missed the Irrlicht Engine. I stumbled across your modern build and fantastic, It's
100% what I am after!!!
Do you plan to implement the Vulkan API any time soon?

Again, thank you so much for your work modernizing such a great engine!

**Comment by CarverNZ** [comment]
May I suggest that we need a method to SetAssetPath inside CBaseApp instead of having a hard coded "BuiltIn" directory being used for developer freedom.
This will make the Engine more portable and easier to set up new projects in the future.
 CarverNZ
